Source/WebCore/bindings/scripts/StaticString.pm

```diff
@@ -29,7 +29,6 @@ use Hasher;
 sub GenerateStrings($)
 {
     my $stringsRef = shift;
-    my %strings = %$stringsRef;
```

Comment:
This change seems unrelated. If it's a cleanup, maybe we want to do that in a separate followup.

Reply:
Will split out.